RecoEgamma/EgammaIsolationAlgos/python/pfClusterIsolationRemap_cfi.py

```diff
@@ -0,0 +1,32 @@
+import FWCore.ParameterSet.Config as cms
+
+electronHcalPFClusterIsolationRemapper = cms.EDProducer('ElectronHcalPFClusterIsolationRemapper',
```

Comment:
this is another example of cfi containing multiple objects and used solely in one cff.
In view of possible other uses, this can be just renamed to a cff.